### DISCUSSION TITLE
C128 With SD2IEC Device Results in File Not Found and Locks Machine

### DISCUSSION THREAD

**Discussion by paulhocker** [frntc/RAD#10] [General]
Starting this thread to have it separate from the other C128 discussion. Perhaps they are related. At any rate, when I press X and any configuration is selected on the RAD is will go through its normal boot sequence but then tries to access the SD2IEC which results in a file not found error. I think it is trying to find a boot sector (I have not tried adding one to the sd card yet) which results in the error. But what is strange is that it seems to lock machine. Sometimes I can get out of the lock doing a RS/Restore, but most times I need to power cycle the C128. I have tried 3 diff sd2iec with same results.

I plan to try with a real CBM device as well as the SD2IEC

Thanks

**Comment by paulhocker** [comment]
This is interesting. I changed the Device number to #9 and it no longer gives me the error. So I do think it is trying to boot something off of the sd2iec.

**Comment by frntc** [comment]
I will see if I can reproduce this, but it'll take a few days before I find the time.

**Comment by paulhocker** [comment]
My C128D does not have the same problem initializing the REU when accessing the internal 1571 so I am pretty sure this is a problem with the SD2IEC device not providing a boot sector. I am going to research this some more but I think you can probably close this unless you want to wait and see what else I discover?
